src/tree_cmd.cpp

```diff
@@ -193,11 +220,16 @@ static void PlaceTreeGroups(uint num_groups)
 			int x = GB(r, 0, 5) - 16;
 			int y = GB(r, 8, 5) - 16;
 			uint dist = abs(x) + abs(y);
+			uint max_dist = (_settings_newgame.game_creation.tree_placer == TP_IMPROVED) ? GB(r, 0, 3) + 8 : MAX_DISTANCE_FROM_CENTER;
+
+			if (dist > max_dist)
```

Comment:
Please either put this on a single line, or add {} around it. This form only leads to errors over time :)

Reply:
I splitted this line, now it's more clear

Reply:
I think TrueBrain meant the coding style for the `if`. Either make it a single line without any linebreaks, or use `{}`.